src/pages/UserInfos/questions/Age.svelte

```diff
@@ -43,13 +43,17 @@ const AGE_QUESTION: { value: Age, label: string }[] = [
 ]
 </script>
 
-<div class="mb-3 row">
-    <label for="" class="col-sm-2 col-form-label">{$_('user_infos.age.what_is_your_age')}:</label>
-    <div class="col-sm-10">
-      <Select 
-        bind:value={age}
-        values={AGE_QUESTION}
-        icon={undefined}
-      />
-    </div>
-</div>
\ No newline at end of file
+<div class="d-flex age">
+    <label class="col-sm-2 col-form-label fs-5">{$_('user_infos.age.what_is_your_age')}:</label>
+    <Select 
+      bind:value={age}
+      values={AGE_QUESTION}
+      icon={undefined}
+    />
+</div>
+
+<style>
+.age {
+  font-weight: 500;
```

Comment:
Would be nice to add and use a `.fw-500` class 👌

Reply:
✅